src/main/java/com/datamountaineer/kcql/Kcql.java

```diff
@@ -190,6 +250,16 @@ public String getTarget() {
     //return new ArrayList<>(fields);
   }
 
+  public List<Field> getKeyFields() {
+    return keyFields;
+    //return new ArrayList<>(fields);
```

Comment:
minor, but we should clean up this commented code (also below)

Reply:
cleaned up